src/app/shared/services/file-manager/file-manager.service.ts

```diff
@@ -48,11 +47,11 @@ export class FileManagerService extends SyncServiceBase {
   }
 
   /**
-   * Save an asset file to device storage.
+   * Save an asset file to device storage, to be accessible to the user.
    * On native devices, save file to native storage and optionally trigger prompt to open with native apps.
    * On web, prompt standard browser download.
    * @param options.relativePath The relative path to an asset, in standard authoring syntax
-   * @param options.open On native devices, open the file after download. Default: true
+   * @param options.open If true, then the file will be opened after download on native devices. Default: true
    */
   private async downloadTemplateAsset(options: {
```

Comment:
nit(non-blocking) Not specifically related to the code but I've seen a few mentions of the `Documents` directory and just wanted to check if this is the folder you want? If using the `Data` directory then on IOS this is a proxy for the Documents directory, but on Android will be local application data directory. I'm not sure what is best (depends on whether you want these files to be accessible to other apps or not) 
https://capacitorjs.com/docs/apis/filesystem#directory

Reply:
Thanks for flagging. This was written before we has iOS integration and so I was only considering the Android case, but the intention was that this method (triggered by the `save_to_device` action) would download the file and make it accessible to other apps.

Those docs are a bit ambiguous to me, each of the possible storage locations is described slightly differently, even when they seem to be referring to the same location, and the only explicit mention that files will be available publicly to other apps is for the combination of `DOCUMENTS` and the Android platform. I think I'll have to handle this with an explicit check for the platform and change the storage directory accordingly – my best guess would be that I want to target `EXTERNAL_STORAGE` on iOS, but I can experiment with which location works for making the file publicly available.